diff --git a/dist/rollup.d.ts b/dist/rollup.d.ts
index ed4545113c6a5f31ea77f7b57368bf4c274ff465..4735dc78f4ee556d685466168cd0be5df2b65f07 100644
--- a/dist/rollup.d.ts
+++ b/dist/rollup.d.ts
@@ -60,7 +60,7 @@ export declare class CompilerState {
  *
  * @public
  */
-export declare const enum ConsoleMessageId {
+export declare enum ConsoleMessageId {
     /**
      * "Analysis will use the bundled TypeScript version ___"
      */
@@ -323,7 +323,7 @@ export declare class ExtractorConfig {
  *
  * @public
  */
-export declare const enum ExtractorLogLevel {
+export declare enum ExtractorLogLevel {
     /**
      * The message will be displayed as an error.
      *
@@ -445,7 +445,7 @@ export declare class ExtractorMessage {
  * Specifies a category of messages for use with {@link ExtractorMessage}.
  * @public
  */
-export declare const enum ExtractorMessageCategory {
+export declare enum ExtractorMessageCategory {
     /**
      * Messages originating from the TypeScript compiler.
      *
@@ -491,7 +491,7 @@ export declare const enum ExtractorMessageCategory {
  *
  * @public
  */
-export declare const enum ExtractorMessageId {
+export declare enum ExtractorMessageId {
     /**
      * "The doc comment should not contain more than one release tag."
      */
